qa/vagrant/src/test/resources/packaging/scripts/30_deb_package.bats

```diff
@@ -63,16 +63,17 @@ setup() {
     dpkg -s 'elasticsearch'
 }
 
-##################################
-# Check that the package is correctly installed
-##################################
 @test "[DEB] verify package installation" {
     verify_package_installation
 }
 
-##################################
-# Check that Elasticsearch is working
-##################################
+@test "[DEB] elasticsearch isn't started by package install" {
+    # Wait a second to give Elasticsearch a change to start if it is going to.
+    # This isn't perfect by any means but its something.
+    sleep 1
+    ! ps aux | grep elasticsearch | grep java
```

Comment:
Ok - gave it a try. You need: ```! sudo -u elasticsearch jps | grep -i elasticsearch``` which is pretty cumbersome and seems more error prone so I'm going to stick with `ps aux`